src/dymaptic.GeoBlazor.Core/Scripts/dotNetBuilder.ts

```diff
@@ -749,3 +752,29 @@ export function buildDotNetTimeInterval(interval: any): any | null {
         value: interval.value
     } as any;
 }
+
+export function buildDotNetFeatureType(result: any) {
+
+    if (!hasValue(result)) {
+        return null;
+    }
+    let dotNetDomains = {};
+    for (var domain in result.domains) {
+        if (Object.prototype.hasOwnProperty.call(result.domains, domain)) {
+
+            dotNetDomains[domain] =  buildDotNetDomain(result.domains[domain]);
+        }
+    }
+
+    return {
+        id: result.id,
+        domains: dotNetDomains,
+        declaredClass: result.declaredClass,
+        name: result.name,
+        templates: result.templates,
```

Comment:
have you validated that `templates` converts OK without a builder method?

Reply:
yes